plugins/home/api-report.md

```diff
@@ -181,6 +197,20 @@ export type LayoutConfiguration = {
   resizable?: boolean;
 };
 
+// @public
+export class LocalStorageVisitsApi extends VisitsApiFactory {
+  // (undocumented)
+  static create(options: LocalStorageVisitsApiOptions): LocalStorageVisitsApi;
+}
+
+// @public (undocumented)
+export type LocalStorageVisitsApiOptions = {
+  localStorage?: Window['localStorage'];
+  randomUUID?: Window['crypto']['randomUUID'];
```

Comment:
These two (and the randomUUID in `CoreStorageVisitsApiOptions` above too) are only used for testing right? Might be nice to hide that from the public surface; end users don't really have to ever set these. Could be part of some internal-only class interface or so perhaps?

Reply:
fixed on 53713d4ee572a070625e76c9684a4f1fc63acf3f